impeller/entity/entity_pass.cc

```diff
@@ -153,33 +154,33 @@ static RenderTarget CreateRenderTarget(ContentContext& renderer,
   /// changed for the lifetime of the textures.
 
   if (context->SupportsOffscreenMSAA()) {
-    return RenderTarget::CreateOffscreenMSAA(
-        *context,                          // context
-        size,                              // size
-        "EntityPass",                      // label
-        StorageMode::kDeviceTransient,     // color_storage_mode
-        StorageMode::kDevicePrivate,       // color_resolve_storage_mode
-        LoadAction::kDontCare,             // color_load_action
-        StoreAction::kMultisampleResolve,  // color_store_action
-        readable ? StorageMode::kDevicePrivate
-                 : StorageMode::kDeviceTransient,  // stencil_storage_mode
-        LoadAction::kDontCare,                     // stencil_load_action
-        StoreAction::kDontCare                     // stencil_store_action
-    );
+    return RenderTargetBuilder()
```

Comment:
You can hoist the builder out of the conditional and set the common options like label, size, etc. and only set the MSAA specific builder options conditionally. That way, you don't have to repeat code in two places. Right now, this is a mechanical replacement of the existing calls.